frontend/src/components/layout/navigation/whatsnew/WhatsNewList.module.scss

```diff
@@ -48,6 +48,12 @@
       padding: $spacing-xs 0;
       cursor: pointer;
 
+      // Forces news icon to be the same size regardless of image size
+      img {
+        width: 50px;
```

Comment:
Hmm, I'd expect the image to have the correct intrinsic size, rather than force-stretching/squeezing them to have the right size? Possibly a `max-width` and `max-height` would be sufficient?

Reply:
Not the first time this happens. Sometimes the image is exported as 60 or 70px by same size. And it renders larger than the other icons. Hence why I added this to make sure we don't run into that. I set them to their `max-` alts. Thanks!